src/hypercore-file-logger.js

```diff
@@ -61,103 +56,34 @@ class HyperCoreFileLogger extends HyperCoreLogger {
     this.republish = republish
     this.pathlike = pathlike
 
-    /** @type {Object<string, Tail>} */
-    this.fileTails = {}
-    /** @type {chokidar.FSWatcher | null} */
-    this.watcher = null
-    /** @type {boolean} */
-    this.isReady = false
+    this.watcher = new FilesWatcher(pathlike, republish, this.feedOpts.valueEncoding)
   }
 
   static getFileDelimiter () {
```

Comment:
do we still need this method?

Reply:
yes, this is necessary to avoid a breaking changes, because somebody can use this method directly since version 0.4.0